documentation/docusaurus.config.js

```diff
@@ -9,7 +9,7 @@ const config = {
   title: 'Iron Bank Manifest Utility',
   tagline: 'Making an update to your IB container just got easier',
   url: 'https://your-docusaurus-test-site.com',
-  baseUrl: '/',
+  baseUrl: '/ib_manifest_util/',
```

Comment:
@kcpevey It seems that this change is breaking the links on the homepage. Although I tried for awhile, I'm not familiar enough with Docusaurus or JS/TS to debug why...

Reply:
If you update the `src: 'ib_manifest_util/img/ib_logo.png'` (line 56) and keep the `baseURL: '/'` you can restore the functionality the links and get the logo to appear.

Reply:
strangely enough the icon was working for me but the links on the front page were, indeed, broken. I made your suggested change and all seems correct now. If the new test actions (included here) had run as part of this PR, it would have caught this.